draft-ietf-quic-ack-frequency.md

```diff
@@ -339,17 +334,32 @@ send an acknowledgement immediately on receiving a reordered ack-eliciting
 packet. This extension modifies this behavior.
 
 If the endpoint has not yet received an ACK_FREQUENCY frame, or if the most
-recent frame received from the peer has an `Ignore Order` value of `false`
-(0x00), the endpoint MUST immediately acknowledge any subsequent packets that
+recent frame received from the peer has a `Reordering Threshold` value that is
+not 0x00, the endpoint MUST immediately acknowledge any subsequent packets that
 are received out of order.
```

Comment:
I don't think this sentence is correct anymore. You only should ack if you received more than X packet out of order. given the default value is 3, I guess you have to immediately send an ack if the new value is 1 or 2 and you have re-order packets that have not be ack'ed yet... not sure how to parse that easily.